bin/test_benchmarks

```diff
@@ -53,4 +57,43 @@ pushd test
     "
 popd || true
 
+# gather auto benchmark data for website
+
+# container size
+container_size=$(docker images \
+  | grep "cyberark\/secretless-broker[^-]" \
+  | awk '{ print $7 }')
+
+echo '<h3>Docker Image Size</h3>
+<p>The Docker image size is auto-updated on this page with every build of the
+project. You may also visit our
+<a href="https://hub.docker.com/r/cyberark/secretless-broker/">DockerHub</a>.</p>' \
+  | tee -a $project_dir/docs/_includes/auto_benchmarks.html
+echo "<p>Docker image size: ${container_size}</p>" \
+  | tee -a $project_dir/docs/_includes/auto_benchmarks.html
+
+echo "Container size: $container_size\n" \
+  | tee -a $project_dir/test/bench.output
+
+# lines of code count
+echo '<h2>Code Metrics</h2>
+<h3>Lines of Code</h3>
+<p>Code line counts were generated with
```

Comment:
This `<p>` tag is opened but I don't see it closed anywhere. Intended?

Reply:
would you believe me if I said yes?